Index: b/kcalc/CMakeLists.txt
===================================================================
--- a/kcalc/CMakeLists.txt
+++ b/kcalc/CMakeLists.txt
@@ -44,7 +44,7 @@
 
 kde4_add_kdeinit_executable( kcalc ${kcalc_KDEINIT_SRCS})
 
-target_link_libraries(kdeinit_kcalc  ${KDE4_KDEUI_LIBS} ${GMP_LIBRARIES})
+target_link_libraries(kdeinit_kcalc  ${KDE4_KDEUI_LIBS} ${GMP_LIBRARIES} ${QT_QTXML_LIBRARY})
 
 install(TARGETS kdeinit_kcalc ${INSTALL_TARGETS_DEFAULT_ARGS})
 
Index: b/kwallet/konfigurator/CMakeLists.txt
===================================================================
--- a/kwallet/konfigurator/CMakeLists.txt
+++ b/kwallet/konfigurator/CMakeLists.txt
@@ -11,7 +11,7 @@
 kde4_add_plugin(kcm_kwallet ${kcm_kwallet_PART_SRCS})
 
 
-target_link_libraries(kcm_kwallet  ${KDE4_KDE3SUPPORT_LIBS}  )
+target_link_libraries(kcm_kwallet  ${KDE4_KDE3SUPPORT_LIBS} ${QT_QTDBUS_LIBRARY}  )
 
 install(TARGETS kcm_kwallet  DESTINATION ${PLUGIN_INSTALL_DIR})
 
Index: b/superkaramba/src/CMakeLists.txt
===================================================================
--- a/superkaramba/src/CMakeLists.txt
+++ b/superkaramba/src/CMakeLists.txt
@@ -83,6 +83,7 @@
 set_target_properties(superkaramba PROPERTIES VERSION ${GENERIC_LIB_VERSION} SOVERSION ${GENERIC_LIB_SOVERSION} )
 
 TARGET_LINK_LIBRARIES(superkaramba
+${QT_QTNETWORK_LIBRARY} ${QT_QTSVG_LIBRARY} ${QT_QTXML_LIBRARY} ${X11_X11_LIB}
 ${KDE4_KIO_LIBS}
 ${KDE4_KPTY_LIBS}
 ${KDE4_KROSSCORE_LIBS}
